delivery_service.rb

```diff
@@ -0,0 +1,37 @@
+class DeliveryService 
+
+  attr_accessor :delivery_confirmed, :distance
+  
+  BIKE_MAX_WEIGHT = 10
+  BIKE_MAX_DISTANCE = 30
+  CAR_MAX_WEIGHT = 100
```

Comment:
These constants should not be here. Data related to bike class belong to bike class. And so on

Reply:
Moved constants